### PR TITLE
Add terminating semicolons to harden syntax

### PR DESCRIPTION
**Pull request by MichaIng**
In the "!function() {" blocks, semicolons were omitted where optional with a trailing line break. This can cause issues when minimising the script. This change adds the trailing semicolon to terminate statements where a removed line break would cause a syntax error and hence hardens the wider use of the script.

In all other parts of the script, terminating semicolons are used consequently, hence this change does not break an agreed coding standard.

Optionally, the semicolon could be added as well when it is not strictly required, e.g. at before closing `}`, which would match the way it is done in all other sections (and other scripts I reviewed).

### CODE REVIEW

**Review by michael-sumner — COMMENTED** [review]
There still appear to be missing semi-colons, would be good to add all the semicolons on here?

**Comment by MichaIng** [comment]
Yes as said, I only added it where a careless minifier would cause syntax errors, hence in case of
```
        return output
    }
```
it was not added, as `return output}` still works fine.

But I agree that adding it consequently as well in such cases does not hurt and aligns the style of the section with the rest of the script and other scripts. Finally removing optional semicolons is probably better a task for minifiers only, while the source code should be consequent, explicit and readable.

Shall I wait for another opinion or add all semicolons now?

**Comment by michael-sumner** [comment]
It would be good to add the semi-colons in this case (so that it's an all-or-nothing) approach to adding semi-colons to keep with consistency

**Comment by MichaIng** [comment]
Okay, done.

**Review by ytbigvinny27 — APPROVED** [review]

**Comment by MichaIng** [comment]
Looping through my open PRs, is something missing here?

**Comment by helen** [comment]
Hello - you will want to open this as a PR against `wordpress/wordpress-develop` and open an accompanying Trac ticket if there isn't one already. We don't merge PRs either way, but that repository is a mirror of our development repo and will run tests, etc. as a part of the review process. This repository is a mirror of the built repo.

**Comment by MichaIng** [comment]
Thanks for the info, I'll do that.

**Comment by MichaIng** [comment]
Okay, easier said than done. Can someone give me a hint where I find the script code that is responsible? https://github.com/WordPress/wordpress-develop seems to be structured differently and I fail to find the particular script or a specific function (like `sprintf_format`) used by it in any of the repositories files.

Or is it a 3rd party/external component?

**Comment by helen** [comment]
I see, this looks like it's a built file, seems to come from https://github.com/alexei/sprintf.js/blob/master/src/sprintf.js via https://github.com/WordPress/gutenberg/blob/trunk/packages/i18n/src/sprintf.js

**Comment by MichaIng** [comment]
Many thanks, yes that is it. Ironically the last commit was about fixing semicolons, but only the one at EOF 😄. I'll hence open a PR over there.

**Pull request closed without merging**
